sdk/monitor/opentelemetry-exporter/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "settings": {
+    "node": {
+      "tryExtensions": [".ts"]
+    }
+  },
+  "extends": [
+    "plugin:node/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier/@typescript-eslint"
+  ],
+  "rules": {
+    "no-underscore-dangle": ["error", { "allowAfterThis": true }],
+    "node/no-unsupported-features/es-syntax": ["error", { "ignores": ["modules"] }],
+    "import/prefer-default-export": "off"
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "./tsconfig.json",
+    "tsconfigRootDir": ".",
+    "sourceType": "module",
+    "createDefaultProgram": true
```

Comment:
I think we found this option made the linter very slow, not sure if that's been your experience

Reply:
I can remove it. Thanks!